src/pudl/extract/eia176.py

```diff
@@ -3,24 +3,40 @@
 The EIA Form 176 archive also contains CSVs for EIA Form 191 and EIA Form 757.
 """
 
-from dagster import asset
+import pandas as pd
+from dagster import AssetsDefinition, asset
 
 from pudl.extract.csv import CsvExtractor, get_table_file_map
 
 DATASET = "eia176"
 
 
-@asset(required_resource_keys={"datastore"})
-def raw_eia176__company(context):
-    """Extract raw EIA company data from CSV sheets into dataframes.
+def eia_176_asset_factory(
```

Comment:
I think we should stick with the dataset ID `eia176` and never introduce an `_` in here. It's nice to be able to search for these IDs reliably and use the same idioms for all the datasets like `eia860`, `eia861`, `eia923` etc.

Reply:
You mean in the function names? Simple enough fix, that makes sense.

Reply:
Function names, variable names, tables, classes, anywhere we're naming something to refer to the dataset.

Reply:
Not sure what we'll want to do about the mish-mash of different forms that were hiding inside this zipfile.